content/en/docs/reference/command-line-tools-reference/feature-gates-removed/index.md

```diff
@@ -31,159 +31,4 @@ In the following table:
-- {{< feature-gate-description name="RetroactiveDefaultStorageClass" >}}
-- {{< feature-gate-description name="RootCAConfigMap" >}}
-- {{< feature-gate-description name="RotateKubeletClientCertificate" >}}
-- {{< feature-gate-description name="RunAsGroup" >}}
-- {{< feature-gate-description name="RuntimeClass" >}}
-- {{< feature-gate-description name="SCTPSupport" >}}
-- {{< feature-gate-description name="ScheduleDaemonSetPods" >}}
-- {{< feature-gate-description name="SeccompDefault" >}}
-- {{< feature-gate-description name="SelectorIndex" >}}
-- {{< feature-gate-description name="ServiceAccountIssuerDiscovery" >}}
-- {{< feature-gate-description name="ServiceAppProtocol" >}}
-- {{< feature-gate-description name="ServiceIPStaticSubrange" >}}
-- {{< feature-gate-description name="ServiceInternalTrafficPolicy" >}}
-- {{< feature-gate-description name="ServiceLoadBalancerClass" >}}
-- {{< feature-gate-description name="ServiceLoadBalancerFinalizer" >}}
-- {{< feature-gate-description name="ServiceLBNodePortControl" >}}
-- {{< feature-gate-description name="ServiceNodeExclusion" >}}
-- {{< feature-gate-description name="ServiceTopology" >}}
-- {{< feature-gate-description name="SetHostnameAsFQDN" >}}
-- {{< feature-gate-description name="StartupProbe" >}}
-- {{< feature-gate-description name="StatefulSetMinReadySeconds" >}}
-- {{< feature-gate-description name="StorageObjectInUseProtection" >}}
-- {{< feature-gate-description name="StreamingProxyRedirects" >}}
-- {{< feature-gate-description name="SupportIPVSProxyMode" >}}
-- {{< feature-gate-description name="SupportNodePidsLimit" >}}
-- {{< feature-gate-description name="SupportPodPidsLimit" >}}
-- {{< feature-gate-description name="SuspendJob" >}}
-- {{< feature-gate-description name="Sysctls" >}}
-- {{< feature-gate-description name="TTLAfterFinished" >}}
-- {{< feature-gate-description name="TaintBasedEvictions" >}}
-- {{< feature-gate-description name="TaintNodesByCondition" >}}
-- {{< feature-gate-description name="TokenRequest" >}}
-- {{< feature-gate-description name="TokenRequestProjection" >}}
-- {{< feature-gate-description name="TopologyManager" >}}
-- {{< feature-gate-description name="UserNamespacesStatelessPodsSupport" >}}
-- {{< feature-gate-description name="ValidateProxyRedirects" >}}
-- {{< feature-gate-description name="VolumePVCDataSource" >}}
-- {{< feature-gate-description name="VolumeScheduling" >}}
-- {{< feature-gate-description name="VolumeSnapshotDataSource" >}}
-- {{< feature-gate-description name="VolumeSubpath" >}}
-- {{< feature-gate-description name="VolumeSubpathEnvExpansion" >}}
-- {{< feature-gate-description name="WarningHeaders" >}}
-- {{< feature-gate-description name="WindowsEndpointSliceProxying" >}}
-- {{< feature-gate-description name="WindowsGMSA" >}}
-- {{< feature-gate-description name="WindowsHostProcessContainers" >}}
-- {{< feature-gate-description name="WindowsRunAsUserName" >}}
+{{< feature-gate-list show-removed="true" >}}
```

Comment:
A HTML guidance comment on how to address any issue(s) related to the description items listed below would be helpful here.

Reply:
How's this?